lib/discover/localpackets.go

```diff
@@ -26,8 +26,8 @@ type Announce struct {
 }
 
 type Relay struct {
-	Address string // max:256
-	Latency int32
+	URL     string `json:"url"` // max:256
```

Comment:
In fact this may even be an `xdr:noencode` or whatever the syntax is for that, as the binary coding is only used for local discovery and I don't think there's ever any need to announce relays there?